src/modules/invitation/invitation.service.ts

```
@@ -0,0 +1,369 @@
+        `Une invitation en attente existe déjà pour l'email "${email}" dans la boutique "${store.name}".`,
+      );
+    }
+
+    // 5. Générer un token unique et hacher
+    const rawToken = uuidv4(); // Utilise UUID pour un token unique
+    const tokenHash = await bcrypt.hash(rawToken, 10); // Hache le token pour le stockage
+
+    const expiresAt = new Date();
+    expiresAt.setDate(
+      expiresAt.getDate() +
+        parseInt(this.configService.jwtRefrehTokenExpiration?.replace('d', '') || '7', 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)